src/components/cards/ListViewTaskCard.tsx

```diff
@@ -52,42 +52,33 @@ export const ListViewTaskCard = ({
           <Box sx={{ paddingTop: '2px', paddingBottom: '2px' }}>
             <Stack direction="row" columnGap={8} alignItems="center" justifyContent="space-between">
               <Stack sx={{ width: '100%', cursor: 'pointer' }} direction="row" alignItems="flex-end" columnGap={4}>
-                <Typography variant="sm" fontWeight={400} sx={{ color: (theme) => theme.color.gray[500] }}>
+                <Typography
+                  variant="sm"
+                  fontWeight={400}
+                  sx={{ color: (theme) => theme.color.gray[500], flexGrow: 0, width: '70px' }}
+                >
                   {task.label}
                 </Typography>
-                <Typography variant="sm">{task?.title}</Typography>
+                <Typography variant="sm" sx={{ width: { xs: '100px', sm: '200px', md: 'auto' } }}>
+                  {task?.title}
+                </Typography>
               </Stack>
-              <Stack
-                direction="row"
-                alignItems="center"
-                columnGap="20px"
-                sx={{
-                  minWidth: {
-                    xs: 'none',
-                    sm: '300px',
-                    md: '350px',
-                  },
-                }}
-              >
+              <Stack direction="row" alignItems="center" justifyContent={'space-between'} columnGap={1}>
```

Comment:
The way we are adding width here to define the element size is wrong here. Instead of width, we should add gaps for the flex containers. For example, see the image below. 
![Screenshot from 2024-07-04 17-39-09](https://github.com/copilot-platforms/tasks-app/assets/38468429/166f3fa3-efbc-4e9a-b38d-e35788263816)

The title is going into next line for the size below sm i.e. 600px. If you see the Figma then we can clearly see the width for title as 135px minimum and the gap is 16px which doesn't match the implementation in this PR. 
In our implementation gap is around 33-34px - 
![Screenshot from 2024-07-04 17-45-53](https://github.com/copilot-platforms/tasks-app/assets/38468429/ffec59a1-da4b-4b07-91d0-7c0d50fe4660)

Gap in Figma - 
![Screenshot from 2024-07-04 17-44-20](https://github.com/copilot-platforms/tasks-app/assets/38468429/5b7b0d0f-87f5-4cd3-a5e1-6d8d75e95afe)

Width in Figma -
![Screenshot from 2024-07-04 17-45-08](https://github.com/copilot-platforms/tasks-app/assets/38468429/a21d2e4a-d15b-4889-849f-0827dfd4d8f8)

Never give a fix width for such elements. Let's give floating width and adjust the spacing using gaps.